### DISCUSSION TITLE
Can I use mmclassification's wandbhook to log a validation loss to wandb?

### DISCUSSION THREAD

**Discussion by khh0106** [open-mmlab/mmclassification#1451] [Q&A] [answered]
I'm curious.
Also, what should I change in the config file to save the weight file to wandb? 
What parameter do I need to fix?

**Comment by Ezra-Yu** [comment, marked as answer]
please refer to the [DOC](https://mmclassification.readthedocs.io/en/1.x/advanced_guides/runtime.html#visualizer)

**Reply by khh0106** [reply]
DOC, but I've tried referring to the.
I don't see any parameter settings for validation loss.
Is there something I'm missing?

**Reply by khh0106** [reply]
thank you, I used wandb hook on cifar100 dataset and checked that metric and train loss are displayed,  
but validation loss is not displayed.  
Is there any additional settings in config that I can make to check it?

**Comment by khh0106** [comment]
I switched to mmPretrain and checked wandb by referring to the linked [documentation](https://mmengine.readthedocs.io/en/latest/common_usage/visualize_training_log.html?highlight=wandb#wandb).  
However, I still don't know how to log the validation loss.

**Reply by Ezra-Yu** [reply]
> However, I still don't know how to log the validation loss.

In fact, it is not possible to display the losses in val phase now. The current code does not calculate the loss in the val and test phases
